specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/appconfiguration.json

```diff
@@ -1723,6 +1763,37 @@
       },
       "readOnly": true
     },
+    "OperationDetails": {
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "status": {
+          "enum": [
+            "NotStarted",
+            "Running",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
```

Comment:
Is there a reason for it to be an extensible enum?

Reply:
```suggestion
            "modelAsString": false,
```